docs/content/direct/pre-reqs.md

```diff
@@ -58,8 +58,10 @@ follow [these instructions](https://github.com/kubestellar/kubeflex/blob/main/do
 ## Additional Software For Building KubeStellar from Source
 
 - [**go**](https://go.dev/doc/install) version 1.21 or higher - to build Kubestellar
-- [**make**](https://www.gnu.org/software/make/) - to build Kubestellar and create the Kubestellar container
-- [**ko**](https://ko.build/install/) - to create the Kubestellar container
+- [**make**](https://www.gnu.org/software/make/) - to build Kubestellar and create the Kubestellar container images
+- [**ko**](https://ko.build/install/) - to create some of the Kubestellar container images
+- **docker** (or equivalent that implements `docker buildx`) - to create other KubeStellar container images
```

Comment:
I'm searching through using keyword `buildx` and see it is used only in [hack/build-kubectl-image](https://github.com/kubestellar/kubestellar/blob/main/hack/build-kubectl-image.sh). Is 'other KubeStellar container images' referring to this usage?

Reply:
yes